src/net/sourceforge/kolmafia/modifiers/DoubleModifier.java

```diff
@@ -664,7 +694,7 @@ public static String parseModifier(final String enchantment) {
         String tag = mod.getTag();
 
         if (matcher.groupCount() == 0) {
-          return tag;
+          return tag + ": 1";
```

Comment:
It would be nice to have a unit test for this some time, but going to approve the PR

Reply:
thanks!

I have more modifier related test changes to do with the tag stuff from #2494, I will add a test for this.